### DISCUSSION TITLE
Question: project.xml file

### DISCUSSION THREAD

**Discussion by Mayu-can** [FunkinCrew/Funkin#1520] [Compiling help]
i have a question.
when im trying to run a test (''lime test windows" in the windows powershell)
It keeps saying "you must have a "project.xml file or specify another valid project file when using"test" comment
i really dont know what to do.
PLEASE HELP.

**Comment by ExpiredOats** [comment]
this happens to me sometimes, what i do is just go to my files and click to where nothing is highlighted in funkin-master.
![Screenshot (5)](https://user-images.githubusercontent.com/82234798/114567309-babd4580-9c38-11eb-8bb8-7bf88ba8e2fe.png)
![Screenshot (6)](https://user-images.githubusercontent.com/82234798/114567316-bc870900-9c38-11eb-8331-042fc12f8ccf.png)
notice how I highlight art, if i go to powershell and try to build the game, itll give me the error. just make sure youre not highlighting anything.

**Comment by Mayu-can** [comment]
Thank you, but now it says that MSVC is not setup automatically do you know how I do that?

**Comment by R-code-0** [comment]
> Thank you, but now it says that MSVC is not setup automatically do you know how I do that?

You installed required programs?
